.github/workflows/test.yml

```
@@ -12,6 +12,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    continue-on-error: true
     strategy:
```

```suggestion
    strategy:
      fail-fast: false
```
Should do this as well so they do not abort eachother